PencilCase/PencilCase/scripts/Navigator.ts

```diff
@@ -80,6 +80,10 @@ export class Navigator {
                 }
             }
         }
+
+        if (parameters.toPage === ('#' + Consts.Pages.HomePage.Id)) {
```

Comment:
When go to home page, need to set the home page as active page.
Because we use the active page's back function as the backbutton event handler.